CHANGES.md

```diff
@@ -33,6 +33,18 @@ namespace, under `cylc.jinja.filters`.
 
 Cylc Review was also removed in this version.
 
+-------------------------------------------------------------------------------
+## __cylc-8.0a3 (2020?)__
+
+Fourth alpha release of Cylc 8.
+
+(See note on cylc-8 backward-incompatible changes, above)
+
+### Enhancements
+
+[#3617](https://github.com/cylc/cylc-flow/pull/3617) - For integer cycling mode
+there is now a default initial cycle point of 1.
```

Comment:
I thought this could go into alpha 0.2?

Reply:
It can, if approved before the other PRs are merged we can bump it up to a2. We were expecting an earlier release...